client/src/components/product_overview/ProductInfo.jsx

```diff
@@ -40,34 +40,37 @@ class ProductInfo extends React.Component {
     };
 
     return (
-      <div id="info">
-        <div className="info">
-          <StarsGlobal value={this.props.starValue} />
+      this.props.expandedView
+        ? null
+        : <div id="info">
+          <div className="info">
+            <StarsGlobal value={this.props.starValue} />
+            <p onClick={() => { const element = document.getElementsByClassName('Reviews')[0]; element.scrollIntoView(); }} className="read_reviews">Read all reviews</p>
+          </div>
+          <div className="product_category info">
+            {this.props.product.category}
+          </div>
+          <div className="product_name info">
+            {this.props.product.name}
+          </div>
+          <div className="product_price info">
```

Comment:
Consider using one className for multiple instances to reduce CSS being spread out so much. Are there multiple elements here that will receive the same styling? Remember class names are for multiple elements and ID's are for unique elements.